bound/kt/src/main/kotlin/web5/sdk/vc/StatusListCredential.kt

```diff
@@ -0,0 +1,27 @@
+package web5.sdk.vc
+
+import web5.sdk.Json
+import web5.sdk.rust.StatusListCredential as RustCoreStatusListCredential
+
+data class StatusListCredential(
```

Comment:
we want this to match the APID so we need to add a `val base VerifiableCredential` here